subworkflows/local/impute_quilt/impute_quilt.nf

```diff
@@ -0,0 +1,48 @@
+include { QUILT_QUILT     } from '../../../modules/nf-core/quilt/quilt/main'
+
+workflow IMPUTE_QUILT {
+
+    take:
+    ch_hap_legend                            // channel: [ val(meta), hap, legend ]
+    ch_input                                //  channel: [ val(meta), bam, bai ]
+    ch_chunks                              //   channel: [ val(meta), start_coordinate, end_coordinate, number ]
+
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    posfile             = []
+    phasefile           = []
+    posfile_phasefile   = [[id: null], posfile, phasefile]
```

Comment:
Why is it all empty ?
Is it not necessary ?

Reply:
These parameters are optional. This first version of QUILT only uses mandatory parameters. See my note in the PR "Future improvements will include: Using optional files (genetic map file, phasefile, posfiles). This requires modification of general params used in the pipeline."